blas/testblas/level1double.go

```diff
@@ -1772,7 +1772,7 @@ var DrotmgTests = []DrotmgTestStruct{
 		Name: "RD1_Big_RD2_Big_Flag_0",
```

Comment:
How are the test cases named? Flag_0 but Rescaling is -1 ...
Also in other test cases.